packages/design-system/src/theme.ts

```diff
@@ -1,13 +1,47 @@
-import { theme as chakraTheme } from '@chakra-ui/core';
+import { DefaultTheme, theme as chakraTheme } from '@chakra-ui/core';
 
-export const theme: typeof chakraTheme = {
+interface MetaColors {
+  dark: string;
+  purpleBoxDark: string;
+  purpleBoxLight: string;
+  purpleTag: string;
+  blueLight: string;
+  cyanText: string;
+}
+
+interface MetaTheme {
+  colors: MetaColors;
+}
+
+type Theme = DefaultTheme & MetaTheme;
+
+export const theme: Theme = {
   ...chakraTheme,
   colors: {
     ...chakraTheme.colors,
+    // metagame colors
+    purple: {
+      50: '#f1e7ff',
+      100: '#cfbafc',
+      200: '#ab8df3',
+      300: '#8460ed',
+      400: '#5a32e6',
+      500: '#4d19cd',
+      600: '#4412a0',
+      700: '#380d74',
+      800: '#260747',
+      900: '#11021d',
+    },
+    dark: '#1B0D2A',
+    purpleBoxDark: '#261943',
+    purpleBoxLight: '#392373',
```

Comment:
IMO these should be using values from the purple color palette. e.g. purpleBoxDark should be defined as purple.800. Or, we should just update the purple palette with these colors and use it directly.